readme.md

```diff
@@ -431,6 +431,7 @@
 - [Vehicle Security and Car Hacking](https://github.com/jaredthecoder/awesome-vehicle-security#readme)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security#readme) - Security of web apps & services.
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking#readme) - The art of unlocking a lock by manipulating its components without the key.
+- [Authentication & Authorization](https://github.com/casbin/awesome-auth) - Processes of verifying who you are and that you have access to something.
```

Comment:
Please link directly to the `#readme` section. I.e., change: `https://github.com/casbin/awesome-auth` to `https://github.com/casbin/awesome-auth#readme`

Reply:
@meitar Fixed.

Reply:
Please approve it, thanks!